### PR TITLE
Add TPU example

### PR DESCRIPTION
**Pull request by rohitgr7**
## What does this PR do?

The TPU notebook linked in README is broken and too old. Added a new one in examples and linked the same in README.

## Before submitting
- [ ] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), Pull Request section?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together? Otherwise, we ask you to create a separate PR for every change.
- [x] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests? 
- [ ] Did you verify new and existing tests pass locally with your changes?
- [ ] If you made a notable change (that affects users), did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)?

## PR review
Anyone in the community is free to review the PR once the tests have passed.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:  

 - [x] Is this pull request ready for review? (if not, please submit in draft mode)
 - [x] Check that all items from **Before submitting** are resolved
 - [x] Make sure the title is self-explanatory and the description concisely explains the PR
 - [x] Add labels and milestones (and optionally projects) to the PR so it can be classified; _Bugfixes should be including in bug-fix release milestones (m.f.X) and features should be included in (m.X.b) releases._
 

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

**Review by Borda — APPROVED** [review]
lgtm

**Review by s-rog — APPROVED** [review]

File: notebooks/README.md [hunk 1]
```diff
@@ -11,4 +11,5 @@ You can easily run any of the official notebooks by clicking the 'Open in Colab'
...
 | **Trainer Flags**        | Overview of the available Lightning `Trainer` flags                                  |      [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/PytorchLightning/pytorch-lightning/blob/master/notebooks/05-trainer-flags-overview.ipynb)      |
-| **94% Baseline CIFAR10** | Establish a quick baseline of ~94% accuracy on CIFAR10 using Resnet in Lightning     |         [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/PytorchLightning/pytorch-lightning/blob/master/notebooks/06-cifar10-baseline.ipynb)         |
+| **TPU Training**        | Training models on TPUs with Lightning                                                 |      [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/PytorchLightning/pytorch-lightning/blob/master/notebooks/06-tpu-training.ipynb)                |
```
Comment (tchaton) [inline notebooks/README.md:14]: Can you provide more details on which models / datasets are being trained on TPUs.

Comment (Borda) [reply notebooks/README.md:14]: @rohitgr7 it seems you are removing the other example, why?

Comment (Borda) [reply notebooks/README.md:14]: I guess it is our example just moved to this repo...

**Review by justusschock — APPROVED** [review]

File: notebooks/README.md (hunk 1 above)
Comment (Borda) [reply notebooks/README.md:14]: @rohitgr7 ^^

Comment (rohitgr7) [reply notebooks/README.md:14]: the same MNIST. updated.

**Comment by rohitgr7** [comment]
@edenlightning mind review?

**Review by williamFalcon — APPROVED** [review]

**Pull request merged**